### PR TITLE
Adding in build command for docker-compose

### PR DESCRIPTION
**Pull request by DarthHater**
I rule and have completed some work on Case Manager that's ready for review!

Adds in build command for `docker-compose`

I noticed we were missing an explicit command for `docker-compose build` so I'm adding this in for people who use `docker-compose` such as @geoffreyyip 

This pull request makes the following changes:
* Small change to `docker-compose`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 web:
   image: colinxfleming/dcaf_case_management
+  build: .
```
Comment (lomky) [inline docker-compose.yml]: I assert this is indeed a `.` :tada:

**Comment by DarthHater** [comment]
In testing this my local `docker-compose` didn't like it, so I played around a bit with #1376 and superseded this PR, I think.

**Comment by colinxfleming** [comment]
Looks like this is overtaken by #1376, so I'll let this hang out until that's merged.

Thanks @lomky and @geoffreyyip for eyeballing this and poking at it with @DarthHater !

**Comment by DarthHater** [comment]
Closed in favor of #1376

**Pull request closed without merging**